odh-dashboard/base/odh-dashboard-crd.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: odhdashboards.core.opendatahub.io
+spec:
+  group: core.opendatahub.io
+  scope: Namespaced
+  names:
+    plural: odhdashboards
+    singular: odhdashboard
+    kind: OdhDashboard
+  versions:
+    - name: v1alpha
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              required:
+                - notebookSizes
+              properties:
+                dashboardConfig:
+                  type: object
+                  properties:
+                    enablement:
+                      type: boolean
+                    disableInfo:
+                      type: boolean
+                    disableSupport:
```

Comment:
Can you add these properties to the dashboardConfig?

 disableClusterManager: 
      type: boolean
 disableTracking: 
      type:boolean
  disableBYONImageStream: 
      type: boolean
  disableISVBadges: 
    type: boolean